### DISCUSSION TITLE
Menubar shift on focus after PV update.

### DISCUSSION THREAD

**Discussion by dbig** [primefaces/community#888] [PrimeVue]
Hello,
After updating PrimeVue (3.13.1  >  3.45.0) I am seeing a positional shift in my menubar items when focus is applied.  Using the menubar component with #start and #end slots.

When a menu item is poked, it's position jumps as far as it can to the left (ltr-reading), without leaving the viewport.

Comparing the rendered code pre- and post-update, I can see focus-related classes and 'rel' params being added to the element.

I cannot locate any mention of this in the forums, here in Git discussions, in the documentation, and see no obvious item(s) in the changelog since 3.13.1.

Can someone provide a sense of clarity regarding this new behaviour?  How can I manage or disable the jump?  Is there new syntax to defeat it?  Any info will be helpful.

Thank you very much,
Doug

**Comment by tugcekucukoglu** [comment]
Hello,

 There are some breaking changes since v3.13.0. It might be related with this issue: https://github.com/primefaces/primevue/issues/4870
